services/QuillLMS/app/services/diagnostics_organized_by_classroom_fetcher.rb

```diff
@@ -28,46 +30,27 @@ def run
       index_of_existing_classroom = classrooms.find_index { |c| c['id'] == record['classroom_id'] }
       name = grouped_name(record)
 
-      next if record['post_test_id'] &&
```

Comment:
I can't figure out why this was necessary to do previously, and in fact it seemed to be causing a bug where activities without post-tests (like the various College Board-related diagnostics) were showing up with duplicates, so I'm removing it. I'll monitor in case we get related bug reports.